### PR TITLE
Add rulebook_activation module and role

### LINKED ISSUE #10: Create Activation module and role
The module should also encompass the disable and enable POST actions possible for the endpoint

Should also include the creation of extra_vars

### LINKED ISSUE #11: Create Activation_restart module and role

### PR DESCRIPTION
**Pull request by Tompage1994**
# What does this PR do?
Adds the rulebook_activation role and module

# How should this be tested?
Eventually CI, otherwise for now something like this works:

```
- name: Create eda rulebook activation
  ansible.builtin.include_role:
    name: infra.eda_configuration.rulebook_activation
  vars:
    eda_rulebook_activations:
      - name: Github Hook 5
        description: Hook to listen for changes in GitHub
        project: EDA_example
        rulebook: git-hook-deploy-rules.yml
        decision_environment: Automation Hub Default Decision Environment
        extra_vars:
          provider: github-local
          repo_url: https://github.com/ansible/ansible-rulebook.git
        enabled: false
```
# Is there a relevant Issue open for this?
resolves #10 
resolves #11 

# Other Relevant info, PRs, etc

### CODE REVIEW

File: changelogs/fragments/activations.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - Added rulebook_activation module
```
Comment (djdanielsson) [inline changelogs/fragments/activations.yml:3]: not sure this is really a minor change

File: roles/rulebook_activation/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  author: "Derek Waters"
+  description: "An Ansible Role to create a rulebook activation in EDA Controller."
+  company: "Red Hat"
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+  license: "GPLv3+"
+
+  min_ansible_version: "2.9"
```
Comment (djdanielsson) [inline roles/rulebook_activation/meta/main.yml]: change to 2.14

File: roles/rulebook_activation/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+galaxy_info:
+  role_name: "rulebook_activation"
+  author: "Derek Waters"
```
Comment (djdanielsson) [inline roles/rulebook_activation/meta/main.yml]: is this correct or did you forget to change to your name?

**Review by djdanielsson — CHANGES_REQUESTED** [review]
few little things

File: changelogs/fragments/activations.yml (hunk 1 above)
Comment (Tompage1994) [reply changelogs/fragments/activations.yml:3]: True. I've now updated each of the other changelog fragments. Either way it doesn't particularly matter as its pre-release still

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**